lib/screens/collection.dart

```diff
@@ -63,25 +63,55 @@ class _CollectionScreenState extends State<CollectionScreen> {
           Expanded(
             child: SizedBox(
               height: 600,
-              child: ListView.builder(
-                  itemCount: beasties.length,
-                  itemBuilder: (context, index) {
-                    return Card(
-                      child: SizedBox(
-                        height: 80,
-                        child: Row(
-                          children: [
-                            const SizedBox(width: 20),
-                            Image.asset(beasties[index].filename),
-                            const SizedBox(width: 50),
-                            Text(
-                              beasties[index].name,
-                              style: const TextStyle(fontSize: 20),
-                            ),
-                          ],
-                        ),
-                      ),
-                    );
+              child: FutureBuilder(
+                  future: BeastieCollectionsData().getCollections(),
+                  builder: (context, AsyncSnapshot snapshot) {
+                    if (!snapshot.hasData) {
+                      return const CircularProgressIndicator();
+                    }
+                    List<BeastieCollection> allCollections = snapshot.data;
+                    // This should be the collection for the particular user but we can't do
+                    // that just yet.
+                    BeastieCollection _beastieCollection = allCollections[0];
+
+                    return FutureBuilder(
```

Comment:
I don't like this nested FutureBuilder but it's the first solution that came to mind. We might want to change how we do this but if it works, who cares lol